sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/DetectedLanguagePropertiesHelper.java

```diff
@@ -32,7 +32,7 @@ public static void setAccessor(
         accessor = detectedLanguageAccessor;
     }
 
-    public static void setScript(DetectedLanguage detectedLanguage, ScriptKind script) {
-        accessor.setScript(detectedLanguage, script);
+    public static void setScriptKind(DetectedLanguage detectedLanguage, ScriptKind scriptKind) {
```

Comment:
Out of curiosity: What was the feedback for renaming `Script` to `ScriptKind`? I do think `ScriptKind` sounds a little better, and I'm wondering if I need to make this change too.

Reply:
In Java, We usually match the return type in the method name unless it has a specific reason. I was trying to align language by using `getScript()` but Java architect suggests following the original naming pattern. Also, I also do feel `getScriptKind` sounds more reasonable.